Package.swift

```diff
@@ -1,26 +1,41 @@
+// swift-tools-version:4.2
```

Comment:
Swift 4.2 builds should be using the 2.x version of OpenSSL because the 1(flat) package structure used in 1.x has been deprecated for system library packages.

Reply:
@djones6 Here is the passing travis build for Swift-JWK-To-PEM https://travis-ci.org/ibm-cloud-security/Swift-JWK-to-PEM/builds/466092144?utm_source=github_status&utm_medium=notification